.github/workflows/wheels.yml

```
@@ -0,0 +1,43 @@
+name: Build-wheels
+on:
+  release:
+  workflow_dispatch:
+  
+jobs:
+  build_wheels:
+    name: Build wheels for cp${{ matrix.python }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python: [37, 38, 39]
+        include:
+          - os: ubuntu-latest
+            cibw_before_all: yum install -y openblas-devel
+            cibw_environment: CBLAS_ROOT=/usr/include/openblas
+          - os: macos-latest
+            cibw_environment: MACOSX_DEPLOYMENT_TARGET=10.14 # Required for C++17 support
+    steps:
+      - uses: actions/checkout@v2
+
+      # Used to host cibuildwheel
+      - uses: actions/setup-python@v2
+
+      - name: Install cibuildwheel
+        run: python -m pip install cibuildwheel==2.1.1
+
+      - name: Build wheels
+        run: python -m cibuildwheel --output-dir wheelhouse
+        env:
+          CIBW_ARCHS: auto64 # Only build for 64-bit platforms
+          CIBW_BUILD: cp${{ matrix.python }}-*
+          CIBW_BEFORE_BUILD: pip install "cmake>=3.14" numpy
```

Are you sure we need to explicitly install NumPy given that it's already specified in setup.py?
```suggestion
          CIBW_BEFORE_BUILD: pip install "cmake>=3.14"
```
As a side note, I think there's an automated way to install dependencies for setup.py itself which we can potentially use to install CMake (although this almost certainly falls outside the scope of the PR).

---

good suggestion, turns out this was necessary b/c the default manylinux image is `manylinux2010`, which numba doesn't ship wheels for. bumped to `manylinux2014` which fixed the issue.

> As a side note, I think there's an automated way to install dependencies for setup.py
Yeah, it's [PEP 518](https://www.python.org/dev/peps/pep-0518/#specification). definitely something we should look at in the future

---

Okay, nice!